docs/usage/dependency-dashboard.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Dependency Dashboard
+description: Learn all about Renovate's Dependency Dashboard
+---
+
+# Introduction
+
+Renovate has a Dependency Dashboard that shows a overview of the state of your repositories' dependencies.
+
+When you turn on the Dependency Dashboard, Renovate will create a new issue on the repository.
+This issue contains a "dashboard" where you can get a overview of the status of all updates.
+
+Having the Dependency Dashboard enabled also allows you to opt-in to different behavior for certain or even all updates with the "Dependency Dashboard Approval" workflow.
+
+## Supported platforms
+
+The Dependency Dashboard requires that the host platforms supports the concept of issues.
+Read [our FAQ, Renovate core features not supported on all platforms](https://docs.renovatebot.com/faq/#renovate-core-features-not-supported-on-all-platforms) to see if your platform can use the Dependency Dashboard feature.
+
+## How to enable the dashboard
+
+<!-- TODO: It might be nice to change our config presets, so we have one to `:enableDependencyDashboard` and one to `:disableDependencyDashboard`. -->
```

Comment:
I noticed we have a preset to enable the dashboard, but we don't have a preset to turn it off. So maybe we want to rejig things so that:

- `:enableDependencyDashboard` -> turns on dashboard
- `:disableDependencyDashboard` -> turns off dashboard (if it's enabled by default like in `v26`)

Don't know if this is something we want/like?